src/main/java/delete_event_use_case/DeleteEventController.java

```diff
@@ -0,0 +1,29 @@
+package delete_event_use_case;
+
+/**
+ * Controller Class.
+ * Invokes the implementations of deleteEvent given by DeleteEventInteractor, where it will be
+ * used to execute a deletion of an event, calling methods from the input boundary.
+ * @author Ricky Fung
```

Comment:
I believe the author should be, Khizer Ahmad, instead of me

Reply:
I can imagine, thank you!